src/workerd/api/performance.c++

```diff
@@ -5,14 +5,16 @@
 #include "performance.h"
 
 #include <workerd/io/io-util.h>
+#include <workerd/io/worker.h>
 
 #include <kj/encoding.h>
 
 namespace workerd::api {
 
-double Performance::now() {
+double Performance::now(jsg::Lock& js) {
   // We define performance.now() for compatibility purposes, but due to Spectre concerns it
   // returns exactly what Date.now() returns.
+  Worker::Isolate::from(js).getLimitEnforcer().markPerfEvent("performance_now"_kjc);
```

Comment:
Nit: would it make sense / be feasible to capture the reference to the LimitEnforcer when the `Performance` instance is created? No biggie, just like to reduce repeated boilerplate